Formula/volt.rb

```diff
@@ -17,6 +17,8 @@ class Volt < Formula
 
   depends_on "go" => :build
 
+  uses_from_macos "vim" => :test
```

Comment:
Could also make runtime dependency (`volt` is a `vim` package manage).

Though, it may be possible to use alternative `vim` installs (e.g. a GUI vim or `neovim`) so not sure about requiring `vim`.